conneg-by-ap/index.html

```diff
@@ -156,9 +156,11 @@ <h2>Motivation</h2>
       <abbr title="Catalogue Service for the Web">CSW</abbr> [[CSW]] protocols.
     </p>
     <p>
-      This document describes content negotiation based on profiles using HTTP protocol. It introduces HTTP headers and
-      non-HTTP methods, such as via QSAs, and defines a general QSA Application Programming Interface
-      (<abbr title="Application Programming Interface">API</abbr>).
+      This document describes an abstract, functions-based, model for content negotiation by profile and gives both an
+      HTTP and a query string argument <em>realization</em> of it for use in different environments. As required by the
+      <a href="https://www.w3.org/2017/dxwg/charter">Dataset Exchange Working Group's Charter</a> (Section 2.3) two
+      reference implementations for each <em>realization</em> are given. Other <em>realizations</em> are considered
```

Comment:
We don't have those reference implementations yet, do we? I'd propose to remove that statement until we have actual implementation evidence and until then just say that two reference implementations are _required_ but not _given_.